G+, AP, YT integrations, APK Mirror site

Madis0 opened the pull request.

manastungare commented:
This is awesome, thanks @Madis0 !

Looks like the tests are failing, though. Could you please check?
https://travis-ci.org/chimbori/lite-apps/builds/176022394

You can also run `./gradlew check` locally before submitting.

manastungare commented on lite-apps/Google+/manifest.json:
```diff
@@ -19,9 +19,17 @@
     },
+	{
+      "name": "__MSG_profile__",
+      "url": "https://plus.google.com/u/0/me"
```
Please remove the "/u/0", it’s only used for multi-login, and is automatically inserted if not already present.

manastungare commented on lite-apps/APK Mirror/manifest.json:
```diff
@@ -0,0 +1,52 @@
+      "url": "http://www.apkmirror.com/uploads/",
+      "name": "Latest Uploads"
+    },
+	{
```
Unindent by 2.

manastungare commented on lite-apps/YouTube/manifest.json:
```diff
@@ -20,15 +20,50 @@
       "name": "Trending",
       "url": "https://www.youtube.com/feed/trending"
     },
+	{
```
Unindent by 3, here and everywhere.

manastungare commented on lite-apps/YouTube/manifest.json:
```diff
@@ -20,15 +20,50 @@
+      "url": "%u",
+      "name": "Open in lite app"
+    }
+  ],
```
Hermit will support sharing through other means. For now, please use Sharing for actions where text content is uploaded to the Lite App & processed directly.

manastungare commented:
Please use square icons at 300x300.

Madis0 commented:
I think I fixed all problems now.

About indents and gradle - I didn't actually open the full project in Android Studio, just edited the files in Notepad++, that might be the cause of the errors.

manastungare commented:
Thanks! I meant to unindent only the openings braces that were misaligned,
not the whole file! But don’t worry, I can take care of that.

On Tue, Nov 15, 2016 at 8:41 AM, Madis <notifications@github.com> wrote:

> I think I fixed all problems now.
>
> About indents and gradle - I didn't actually open the full project in
> Android Studio, just edited the files in Notepad++, that might be the cause
> of the errors.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/chimbori/lite-apps/pull/2#issuecomment-260694762>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAU_JcFCmHR224RNOcSZW7WHg2JDQbX_ks5q-eDWgaJpZM4KyTW1>
> .
>

manastungare commented:
08:46:25.558 [DEBUG] [TestEventLogger] com.chimbori.liteapps.ValidatorTest > testAllManifestsAreValid FAILED
08:46:25.559 [DEBUG] [TestEventLogger]     java.lang.AssertionError: File [APK Mirror] is missing the field [manifest_url]

Madis0 commented:
Added manifest URL, added to lite-apps.json and tested everything with Android Studio (found out that I could've used gradle bat too but oh well). Hope everything's right now!

manastungare commented:
Awesome, thanks for sending this in!
